app/ide-desktop/lib/dashboard/src/pages/dashboard/Dashboard.tsx

```diff
@@ -330,14 +338,24 @@ export default function Dashboard(props: DashboardProps) {
           }
         },
       }),
-    [
-      inputBindings,
-      /* should never change */ modalRef,
-      /* should never change */ localStorage,
-      /* should never change */ updateModal,
-    ]
+    [inputBindings, modalRef, localStorage, updateModal, setPage]
   )
 
+  React.useEffect(() => {
+    if (detect.isOnElectron()) {
```

Comment:
what about in the browser?

Reply:
but `Cmd+[` and `Cmd+ArrowLeft` do not - and i think users would be quite surprised when they see it in the shortcuts configuration page but somehow they do not work

Reply:
afaict `cmd+[` is not a native browser shortcut (unless it is on macos).
similarly `cmd+arrowleft` is a native browser shortcut on neither windows nor linux